src/components/Clock.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  name: string,
+};
+
+function toNormalDate(date: Date) {
```

Comment:
function names should start with a verb
```suggestion
function normalizeDate(date: Date) {
```